src/schemas/firebolt-openrpc.json

```diff
@@ -302,6 +314,29 @@
                 }
             }
         },
+        "TemporalSetMethod": {
+            "type": "object",
+            "properties": {
+                "tags": {
+                    "allOf": [
+                        {
+                            "type": "array",
+                            "minItems": 1,
+                            "maxItems": 1,
+                            "items": {
+                                "type": "object"
```

Comment:
Are we saying to use temporal sets, you must use an object (not, say, just strings)? Probably fine, but we should document somehow.